mlrun/projects/pipelines.py

```diff
@@ -72,6 +72,7 @@ def __init__(
         ttl=None,
         args_schema: dict = None,
         schedule: typing.Union[str, mlrun.api.schemas.ScheduleCronTrigger] = None,
+        overwrite: bool = None,
```

Comment:
because we are overwriting the whole scheduled workflow, not only the crontab. we used `overwrite_schedule` in the CLI to prevent confusing it with overwriting a project.

Reply:
if @Tankilevitch , @theSaarco and @yaronha agrees, I'm on board as well.